code/game/area/areas.dm

```diff
@@ -62,6 +62,7 @@
 	var/station_area = 0
 	var/centcomm_area = 0
 	var/has_weird_power = FALSE	// If TRUE, SSmachinery will not use the inlined power checks and will call powered() and use_power() on this area.
+	var/no_events = 1 //This area is not affected by station events.

```

Comment:
Unused variable. Remove or use instead of the `if (istype(A, ...))` in the `random_station_area` proc.